tests/expectations/core/test_expect_column_values_to_be_of_type.py

```diff
@@ -117,3 +120,69 @@ def test_expect_column_values_to_be_in_set_render_performance():
 
     duration_s = timeit.timeit(x.render, number=1)
     assert duration_s < 1, f"Rendering took {duration_s} seconds"
+
+
+@pytest.mark.databricks
+@pytest.mark.external_sqldialect
```

Comment:
What does the `external_sqldialect` mark do? Should we test with databricks, postgres, and snowflake?